src/fheroes2/battle/battle_interface.cpp

```diff
@@ -2942,11 +2973,27 @@ void Battle::Interface::HumanBattleTurn( const Unit & unit, Actions & actions, s
             boardActionIntentUpdater.setIntent( { themes, index_pos } );
 
             if ( le.MouseClickLeft( battleFieldRect ) ) {
-                MouseLeftClickBoardAction( themes, *cell, boardActionIntentUpdater.isConfirmed(), actions );
+                bool isConfirmed = boardActionIntentUpdater.isConfirmed();
```

Comment:
This variable can be `const`.

```suggestion
                const bool isConfirmed = boardActionIntentUpdater.isConfirmed();
```